content/browser/android/content_startup_flags.cc

```diff
@@ -26,6 +31,21 @@ void SetContentCommandLineFlags(bool single_process) {
 
   base::CommandLine* parsed_command_line =
       base::CommandLine::ForCurrentProcess();
+#if defined(CASTANETS)
+  base::CommandLine::ForCurrentProcess()->AppendSwitch(
+      service_manager::switches::kNoSandbox);
+  base::CommandLine::ForCurrentProcess()->AppendSwitch(switches::kNoZygote);
+  base::CommandLine::ForCurrentProcess()->AppendSwitch(
+      switches::kDisableGpuCompositing);
+  base::CommandLine::ForCurrentProcess()->AppendSwitchASCII(
+      switches::kNumRasterThreads, "4");
+  base::CommandLine::ForCurrentProcess()->AppendSwitchASCII(
+      switches::kLang, "en-US");
+  base::CommandLine::ForCurrentProcess()->AppendSwitchASCII(
+      switches::kEnableLogging, "stderr");
```

Comment:
Need stderr for Android?

Reply:
Seems stderr is not required explicitly for android. I have removed the same in latest commit.